### DISCUSSION TITLE
Acknowledgements and notes on v0.0.7 🚀

### DISCUSSION THREAD

**Discussion by AntonOsika** [AntonOsika/gpt-engineer#401] [Announcements]
First of all. Shoutouts for great contributions:
- Patillacode / @ganktopplease for very valuable execution on maintaining github. Patilla is travelling - if anyone is interested in helping out here: let us know.
- @ori_ash and @cardynal for great work on setting up community guidelines and structure for the community
- @mojofojo for driving documentation for the codebase – looking forward to have it up

Also I want to acknowledge all of you that engage in constructive discussions. This is what drive progress! You know who you are, all of you count. (some of you that come to mind @unclematt @zovos @_thekidd)

Now, over to exciting stuff:
v0.0.7 is live :rocket:

We've focused further on benchmarking **what gpt-engineer can handle** right now.
If you run it with v0.0.7 you'll notice. Will share more updates with learnings from this!

More announcements:
- We have had a few bugs on main. I think they should all be sorted out by now! **If you have problems, please create an issue or even DM me (if you made it work previously but it recently broke)**. We will do our utmoust to not have bugs on main going forward.
**- Please update to the most recent version!**
  - If you are on **development**: `git pull` (commit / `git stash` first if you have uncommitted changes)
  - If you are on **stable**: `pip install --upgrade gpt-engineer`
  
 
**Finally note**

As AI capabilities continue to accelerate, I forsee that our community can become the epicenter for collaborative and open "red team":ing new approaches for dangerous AI Agent capabilities.

This means that we should continue to focus on benchmarks, and be responsible and share our findings when we see that we and others can create tools that can be used with malicious intent.

When things do become dangerous, we should be careful with releasing technology to the world. While still help us devs to build our personal AI code generation toolboxes.

**Comment by venky81060** [comment]
Good evening sir
I am very poor at coding
Could you please give me some instructions how to write and grow up in career
